capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -122,46 +134,85 @@ class ClubPage extends Component {
-          {owner}
-        </Row>
-        <div className='description'> {this.state.club.description} </div>
-        {bookTile}
-        {isOwner &&
-          <SearchBookModal
-            objectId={this.props.id}
-            update={this.handleBookChange}
-            text='Change the Club&quot;s Book'
-            putURL='/api/clubs'
-            type='club'
-            btnStyle='btn btn-primary mb-4 mt-4 mr-2' />
-        }
-        {assignments}
-        {isOwner &&
-          <Form onSubmit={this.handleAssignmentPost} id='assignment-post-form'>
-            <Form.Group controlId='formPostAssignment'>
-              <Form.Label> Post a new assignment! </Form.Label>
-              <Form.Control as='textarea' rows='3' placeholder='Enter assignment text...' />
-            </Form.Group>
-            <Button variant='primary' type='submit'> Submit </Button>
-          </Form>
+      <div >
+
+        {this.state.fetchData ?
+          (<div className="text-center mt-4">
+            <Spinner variant="primary" animation="border" role="status" />
+          </div>)
+          :
+          (
+            <>
+              <Row>
+                <Col>
+                  <h2 className="ml-2">{this.state.club.name}</h2>
+                  {isOwner ?
+                    <h5 className="mb-1 ml-2 text-muted">Owner</h5>
+                    :
+                    <h5 className="mb-1 ml-2 text-muted">Member</h5>}
+                </Col>
+                {isOwner ?
+                  <Col className="m-auto p-0 mr-3">
+                    <div id="modal-buttons" className="mx-2">
+                      <Link to={`/adminclubpage/${this.state.club.id}`}>
+                        <Button className='admin-button my-auto' variant='secondary'>
+                          Admin page
+                        </Button>
+                      </Link>
+                      <SearchUserModal
+                        text='Search/View Members'
+                        checkoutText='Current Members'
+                        btnStyle="btn btn-primary mx-3 my-aut" />
+                      <SearchBookModal
+                        objectId={this.props.id}
+                        update={this.handleBookChange}
+                        text='Change the Club&quot;s Book'
+                        putURL='/api/clubs'
+                        type='club'
+                        btnStyle='btn btn-primary my-auto mr-2' />
+                    </div>
+                  </Col>
+                  :
+                  <Col className="m-auto p-0 mr-3">
+                    <div id="modal-buttons" className="mx-2">
+                      <SearchUserModal
+                        userType='viewer'
+                        text='View Members'
+                        checkoutText='Current Members'
+                        btnStyle="btn btn-primary mx-3 my-aut" />
+                    </div>
+                  </Col>}
+              </Row>
```

Comment:
nit: Mixture of single and double quotes throughout this file, it would be best to make them all single quotes for consistency

Reply:
Ok.